src/patternfly/components/Tabs/tabs.scss

```diff
@@ -578,6 +612,7 @@ $pf-c-tabs--spacer-map: build-spacer-map("none", "sm", "md", "lg", "xl", "2xl");
   position: relative;
   display: flex;
   flex: 1;
+  align-items: center;
```

Comment:
While this _should_ be ok... who knows if it could cause an issue with something someone is doing. Another way to do it would be to add `align-self: center` to the toggle icon element, assuming this declaration is to center that icon.